docs/error-reporting/project-setup/symbolication.md

```diff
@@ -72,162 +67,128 @@ Backtrace supports the dSYM format.
 
-### Generating Symbol Access Tokens
+<b>ELF file with debug info </b>
 
-You can generate a symbol access token to upload the debug symbols for your project. To generate a symbol access token:
+To generate an ELF file with debug info, refer to compiler specific documentation. For example, this can be done with the -g switch in GCC. 
 
-1. Go to **Project Settings > Symbols > Access tokens**.
-1. To generate the token, click **+**.
-1. Copy the token and save it for later.
+<b>SYM file via Breakpad</b>
 
-:::note
-API tokens, under **Project Settings** > **API tokens**, with `symbol:post` capabilities, can also be used to upload symbols. 
-:::
+Note that this approach should only be taken if it is impossible to provide the original binaries. Using .sym’s will result in less detailed information. 
 
-### Breakpad and Socorro
+To generate a .sym, you can use Breakpad’s dump_syms. Assuming the library in question is /lib/libfoo.so and its debugging symbol is /usr/debug/lib/libfoo.so, run: dump_syms /lib/libfoo.so /usr/debug/lib /tmp/libfoo.so.sym.
+You can find additional info in [Breakpad’s documentation](https://www.chromium.org/developers/decoding-crash-dumps/).
 
-Backtrace is compatible with existing Breakpad and Socorro integrations. You can upload the .sym files through an [HTTP POST](#http-api) or with Breakpad's `sym_upload` tool. You must provide a `token` (referring to your symbol access token) and a `format` query string parameter with the value `symbols`.
+</TabItem>
+</Tabs>
 
-Below is an example of a symbol upload using the `sym_upload` tool.
+## Generate symbol access tokens
+Symbol access tokens are used to upload debug symbols. Go to Project Settings > Project > API Tokens to generate a token with the symbol:post capability. Additional details are available in [Creating API Tokens and Submission URLs](https://docs.saucelabs.com/error-reporting/project-setup/submission-url/#creating-api-tokens).
 
-```
-sym_upload null_read_av.sym 'https://submit.backtrace.io/{your-subdomain}/{symbol-access-token}/symbols'
-```
+## Upload to Backtrace
 
-To generate a symbol access token, see [Generating a Symbol Access Token](#generating-a-symbol-access-token).
+### UI
+Navigate to Project settings > Symbols > Overview to review your symbol uploads, 
+including upload history, search functionality, symbol access tokens and more. Backtrace recommends uploading archives (a .tar.gz or .zip file containing one or more .sym ,.pdb , ELF or dSYM files) of symbols for every build you expect to see crashes for. These symbol files can be uploaded manually or hooked up into your build system so they are automatically uploaded.
```

Comment:
Maybe:
"These symbol files can be uploaded manually or by build automation scripts."